Documentation/dev-guide/interacting_v3.md

```diff
@@ -8,6 +8,8 @@ By default, etcdctl talks to the etcd server with the v2 API for backward compat
 export ETCDCTL_API=3
 ```
 
+**However note** that any key that was created using the v2 API will not be able to be queried via the v3 API.  A v3 API ```etcdctl get``` of a v2 key will exit with 0 and no key data, this is the expected behaviour.
```

Comment:
I think this fits better on the tail of the `By default` paragraph above and it won't need the `**`s.